crates/pixi_manifest/src/task.rs

```diff
@@ -116,7 +116,7 @@ impl Task {
     /// Returns the command to execute as a single string.
     pub fn as_single_command(&self) -> Option<Cow<str>> {
         match self {
-            Task::Plain(str) => Some(Cow::Borrowed(str)),
+            Task::Plain(str) => Some(Cow::Owned(str.split(" ").map(|t| quote(t)).join(" "))),
```

Comment:
This change worries me, as in what consequences it has. Does this not break things like this?:
```toml
task = "echo 'hello there'"
```